stackrox/base/policy/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,82 @@
+# Source: stackrox-central-services/templates/01-central-02-security.yaml
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
```

Comment:
podsecurity policies are a clusterwide resource and should be included within the `cluster-scope` folder, much like how the `clusteroles` and `clusterrolebindings` were organized